feat(em): add info how to reset admin password windows (#2532)
scaleway/docs-content: bare-metal/elastic-metal/troubleshooting/reset-admin-password-windows-server.mdx

---
meta:
  title: Resetting the Administrator password on servers running Windows Server
  description: Resetting the Administrator password on servers running Windows Server
content:
  h1: Resetting the Administrator password on servers running Windows Server
  paragraph: Resetting the Administrator password on servers running Windows Server
tags: windows resuce password
categories:
  - bare-metal
  - elastic-metal
dates:
  validation: 2023-12-14
  posted: 2023-12-14
---

> During the installation or reinstallation of a Windows Server operating system, an administrator password is configured by you (Administrator account).

Author: ```suggestion
During the installation or reinstallation of a Elastic Metal server with a Windows Server operating system, an administrator password is set by you (Administrator account).
```

In case you have lost this password, you can reset it via rescue mode.

<Macro id="iam-requirements" />

<Message type="requirement">
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
  - You have [created](/bare-metal/elastic-metal/how-to/create-server) and [installed](/bare-metal/elastic-metal/how-to/install-server) an Elastic Metal server
</Message>

## Removing the old Administrator password

1. Reboot your Elastic Metal server into [rescue mode](/bare-metal/elastic-metal/how-to/use-rescue-mode/).
2. Log into rescue mode.
3. Install the packages `ntfs-3g` and `chntpw` using the APT package manager.
    ```
    sudo apt install ntfs-3g chntpw
    ```
4. Mount the Windows system partition.
    ```
    sudo ntfsfix /dev/sda2
    ```
5. List the user accounts configured in Windows:
    ```
    sudo chntpw -l /mnt/Windows/System32/config/SAM
    ```

>     An output as follows displays:

Reviewer: ```suggestion
    An output similar to the following displays:
```

>     An output as follows displays:

Reviewer 2: ```suggestion
    An output displays as follows:
```

    ```
    chntpw version 1.00 140201, (c) Petter N Hagen
    Hive </mnt/Windows/System32/config/SAM> name (from header): <\SystemRoot\System32\Config\SAM>
    ROOT KEY at offset: 0x001020 * Subkey indexing type is: 686c <lh>
    File size 65536 [10000] bytes, containing 7 pages (+ 1 headerpage)
    Used for data: 343/38728 blocks/bytes, unused: 32/6104 blocks/bytes.

    | RID -|---------- Username ------------| Admin? |- Lock? --|
    | 01f4 | Administrator                  | ADMIN  | dis/lock |
    | 01f7 | DefaultAccount                 |        | dis/lock |
    | 01f5 | Invit�                         |        | dis/lock |
    | 01f8 | WDAGUtilityAccount             |        | dis/lock |
    ```
    <Message type="tip">
      The administrator account of your server can either be `Administrator` or `Administrateur` depending on the installation.
    </Message>
6. Edit the `Administrator` account by running the following command:
    ```
    sudo chntpw -u Administrator /mnt/Windows/System32/config/SAM
    ```

>     An menu as follows displays:

Reviewer: ```suggestion
    A menu displays:
```

>     An menu as follows displays:

Reviewer 2: ```suggestion
    A menu displays as follows:
```

    ```
    ================= USER EDIT ====================

    RID     : 0500 [01f4]
    Username: Administrator
    fullname:
    comment : User account with administrative privileges
    homedir :

    00000220 = Administrators (which has 1 members)

    Account bits: 0x0010 =
    [ ] Disabled        | [ ] Homedir req.    | [ ] Passwd not req. |
    [ ] Temp. duplicate | [X] Normal account  | [ ] NMS account     |
    [ ] Domain trust ac | [ ] Wks trust act.  | [ ] Srv trust act   |
    [ ] Pwd don't expir | [ ] Auto lockout    | [ ] (unknown 0x08)  |
    [ ] (unknown 0x10)  | [ ] (unknown 0x20)  | [ ] (unknown 0x40)  |

    Failed login count: 5, while max tries is: 0
    Total  login count: 2

    - - - - User Edit Menu:
    1 - Clear (blank) user password
    2 - Unlock and enable user account [probably locked now]
    3 - Promote user (make user an administrator)
    4 - Add user to a group
    5 - Remove user from a group
    q - Quit editing user, back to user select
    Select: [q] >
    ```

> 7. Type `1` and press enter to clear the password of the user:

Reviewer: ```suggestion
7. Type `1` and press enter to clear the user password:
```

    ```
    Select: [q] > 1
    Password cleared!
    ```

> 8. Type `q` and press enter to quit editiong the user. A summary of the files edited displays:

Reviewer: ```suggestion
8. Type `q` and press `Enter` to quit. A summary of the files edited displays:
```

> 8. Type `q` and press enter to quit editiong the user. A summary of the files edited displays:

Reviewer 2: ```suggestion
8. Type `q` and press enter to quit editing the user. A summary of the edited files displays:
```

    ```
    Hives that have changed:
    #  Name
    0  </mnt/Windows/System32/config/SAM>
    Write hive files? (y/n) [n] :
    ```

> 9. Type `y` and press enter to validate to write the files.

Reviewer: ```suggestion
9. Type `y` and press `Enter` to confirm the writing of the files.
```

    ```
    Write hive files? (y/n) [n] : y
    0  </mnt/Windows/System32/config/SAM> - OK
    ```
10. Reboot the server back into [normal mode](/bare-metal/elastic-metal/how-to/use-rescue-mode/#how-to-reboot-into-normal-mode).

## Setting a new Administrator password

> 1. [Activate remote access](https://www.scaleway.com/en/docs/bare-metal/elastic-metal/how-to/activate-remote-access/) on your server to be able to connect to it using the KVM over IP device.

Reviewer: ```suggestion
1. [Activate remote access](/bare-metal/elastic-metal/how-to/activate-remote-access/) on your server to be able to connect to it using the KVM over IP device.
```

2. Open the remote console to visualize the screen output of your machine.

> 3. Open the start menu and click on the the `Administrator` user name to open the user settings.

Reviewer: ```suggestion
3. Open the start menu and click the `Administrator` username to open the user settings.
```
